.npmignore

```diff
@@ -5,14 +5,14 @@
 /resources/
 /src/
 /test/
-/typings/
 *.js.map
 *.fixtures.js
 *.mocha.js
 *.mock.js
 *.ts
 *.tgz
 *.iml
+.release-it.json
```

Comment:
Not related to your changes but there is a bug in https://github.com/remerge/turnilo/blob/master/src/client/utils/wrapping-list-index/wrapping-list-index.ts#L4 I found during testing. It should be `newIndex >= listLength`. Could you fix that real quick 😸